packages/client/src/containers/CreateTeamPage/CreateTeam.js

```diff
@@ -0,0 +1,177 @@
+import React, { useState } from 'react';
+import './CreateTeam.css';
+
+function CreateTeam() {
+  const [teamName, setTeamName] = useState('');
+  const [firstName, setFirstName] = useState('');
+  const [lastName, setLastName] = useState('');
+  const [email, setEmail] = useState('');
+  const [teamMembers, setTeamMembers] = useState([]);
+  const employees = [
+    { id: 1, firstName: 'John', lastName: 'Doe', email: 'john@example.com' },
+    { id: 2, firstName: 'Jane', lastName: 'Smith', email: 'jane@example.com' },
+    {
+      id: 3,
+      firstName: 'Alice',
+      lastName: 'Johnson',
+      email: 'alice@example.com',
+    },
+  ];
+
+  const handleInputChange = (inputType, e) => {
+    if (inputType === 'firstNameInput') {
+      setFirstName(e.target.value);
+    } else if (inputType === 'lastNameInput') {
+      setLastName(e.target.value);
+    } else if (inputType === 'emailInput') {
+      setEmail(e.target.value);
+    }
+  };
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+
+    if (!teamMembers.length) {
+      // eslint-disable-next-line no-alert
+      alert('Please add at least one member to the team.');
+    }
+  };
+
+  const handleAddNewMember = () => {
+    const existingTeamMember = teamMembers.find(
+      (member) => member.email === email,
+    );
+
+    if (existingTeamMember) {
+      // eslint-disable-next-line no-alert
+      alert('The member already exist in the team');
+      return;
+    }
+
+    if (!firstName || !lastName || !email) {
+      // eslint-disable-next-line no-alert
+      alert('Please input first name, last name & email');
+    } else {
+      const newMember = {
+        id: null,
+        firstName,
+        lastName,
+        email,
+      };
+
+      teamMembers.push(newMember);
```

Comment:
When adding new members to the team, you directly manipulate the DOM with `teamMembers.push()`. React prefers immutability, so it's better to create a new array and then update the state.

Reply:
`teamMembers.push(newMember)` instead of doing this, you can do this: 
```suggestion
setTeamMembers(prevTeamMembers => [...prevTeamMembers, member]);
```

Reply:
done